QuickTests/FunctionalTests/DescribeTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+@testable import Quick
+import Nimble
+
+class DescribeTests: XCTestCase {
+    
+    func testDescribeErrorsIfExampleGroupsAreNotInstantiated() {
+        
+        // fiddling with global state in this test to test a singleton, what could go wrong?
+        let exampleGroup = World.sharedWorld.currentExampleGroup
+        World.sharedWorld.currentExampleGroup = nil
```

Comment:
Seems like `QuickSpec` specs don't clean up their `World` state after running, they just rely on other specs to totally overwrite it. So in this case `describe` is still valid based on the last spec's `World` state and won't properly error here (or anywhere else) unless `World` has never been initialized by any spec, i.e. in a pure `XCTestCase` environment or a mixed environment before any `QuickSpec`s have run. I'll dig around a bit more and try to find a better way--manually mucking up `World` state and then testing for failure seems like too much of a self-fulfilling prophecy to be a good unit test.